applications/vessel-history/src/features/profile/Profile.tsx

```diff
@@ -51,6 +51,13 @@ const Profile: React.FC = (props): React.ReactElement => {
 
   const [activeTab, setActiveTab] = useState<Tab | undefined>(tabs?.[0])
 
+  const defaultPreviousNames = useMemo(() => {
+    return `+${vessel?.history.shipname.byDate.length} previous ${t(
+      `vessel.names` as any,
+      'names'
```

Comment:
Check https://www.i18next.com/translation-function/plurals if you need to show the translation of "Name" vs "Names"

Reply:
Fixed, thanks!